### ISSUE TITLE
New Feature: Report on Cost Savings

### ISSUE THREAD

**Issue by benburich** [microsoft/startstopv2-deployments#84]
I would love to see a feature that estimates the cost savings from the VMs being powered off. Ideally, It would look great on the dashboard. You could also generate a monthly email to show how much money you have saved the organization. I'm going to work on this myself but let me know if this is in the roadmap. Thanks!

**Comment by ajdinzutic** [comment]
yes, that would be great also to show internally or to the clients the benefits.
